.gitleaks.toml

```
@@ -0,0 +1,10 @@
+title = "MBI Scan"
+# Only using custom rules
+
+[extend]
+useDefault = true
```

Maybe we'll have a pre-commit function in place for toml one day. Ahead of that, I've just been using the `taplo` cli+lsp for sanity checking. Based on the `[[rules]]` formatting below, it looks like we _might_ prefer something like `indent_entries=true` in e.g. `taplo fmt --option indent_entries=true .gitleaks.toml`

```suggestion
  useDefault = true
```